### ISSUE TITLE
Unable to align

### ISSUE THREAD

**Issue by diff7** [MontrealCorpusTools/Montreal-Forced-Aligner#237]
mfa align ../ljspeech_dataset ./fs_two/lexicon/librispeech-lexicon.txt english ../ljspeech_dataset_processed/LJSpeech
All required kaldi binaries were found!
/root/Documents/MFA/ljspeech_dataset/align.log
WARNING - WARNING: Using old temp directory, this might not be ideal for you, use the --clean flag to ensure no weird behavior for previous versions of the temporary directory.
INFO - Setting up corpus information...
INFO - Number of speakers in corpus: 1, average number of utterances per speaker: 13100.0
INFO - Number of speakers in corpus: 1, average number of utterances per speaker: 13100.0
INFO - Parsing dictionary without pronunciation probabilties without silence probabilties
INFO - Creating dictionary information...
INFO - Setting up training data...
Traceback (most recent call last):
  File "/opt/conda/envs/aligner/bin/mfa", line 8, in <module>
    sys.exit(main())
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/command_line/mfa.py", line 339, in main
    run_align_corpus(args, unknown, acoustic_languages)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/command_line/align.py", line 159, in run_align_corpus
    align_corpus(args, unknown_args)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/command_line/align.py", line 91, in align_corpus
    a = PretrainedAligner(corpus, dictionary, acoustic_model, align_config,
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/aligner/pretrained.py", line 59, in __init__
    super(PretrainedAligner, self).__init__(corpus, dictionary, align_config, temp_directory,
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/aligner/base.py", line 58, in __init__
    self.setup()
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/aligner/pretrained.py", line 78, in setup
    super(PretrainedAligner, self).setup()
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/aligner/base.py", line 62, in setup
    self.corpus.initialize_corpus(self.dictionary)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/corpus/align_corpus.py", line 588, in initialize_corpus
    self.split(dictionary)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/corpus/align_corpus.py", line 580, in split
    self._split_texts(split_dir, dictionary)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/corpus/align_corpus.py", line 571, in _split_texts
    save_groups(self.grouped_text(dictionary), directory, pattern)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/corpus/align_corpus.py", line 466, in grouped_text
    lookup = dictionary.split_clitics(t)
  File "/opt/conda/envs/aligner/lib/python3.8/site-packages/montreal_forced_aligner/dictionary.py", line 271, in split_clitics
    initial, final = m.groups()
AttributeError: 'NoneType' object has no attribute 'groups'

**Comment by mmcauliffe** [comment]
So I don't know exactly what's causing this issue, but I changed it in the repo to use a better way of splitting that supports multiple apostrophes (in case that was the issue).  You can try installing following downloading (`python setup.py install` from within the downloaded repo) and see if it fixes it. 

There also looks to be a corpus parsing issue with only one speaker for 13,100 files, so you might want to see about breaking them up into speaker directories or specifying speaker prefixes with the `-s` flag.  Speakers are used for breaking up files for multiprocessing and for speaker adaptation.

**Comment by jah238** [comment]
I had this issue, also, when trying to run `mfa validate` on a `.wav`/`.TextGrid` pair.  The duration of the `.wav` is ~46 minutes.  The `Textgrid` has 2 tiers (1 for each speaker) and the transcriptions use LDC transcription guidelines.  Because of this, I wondered if instances of `--` were causing the error.  (I used MFA 2.0 to align other datasets in prosodylab format and didn't encounter this error.)  Removing `--` didn't resolve the error, and there didn't appear to be any instances of double apostrophe.

All that said, updating to the new dictionary.py from @mmcauliffe did resolve the issue for me.

Updating dictionary.py seemed to break `Annotator`, so I also needed to update the `gui` directory from the repo.
